scipy/optimize/_numdiff.py

```diff
@@ -220,6 +229,9 @@ def approx_derivative(fun, x0, method='3-point', rel_step=None, f0=None,
         Each bound must match the size of `x0` or be a scalar, in the latter
         case the bound will be the same for all variables. Use it to limit the
         range of function evaluation.
+
+        Bounds checking not implemented for the case `return_linear_operator`
```

Comment:
Are you sure blank lines will be rendered correctly by sphinx? (I don't remember).